test/polyrith.lean

```diff
@@ -504,7 +504,7 @@ example (a b c : ℤ) (h1 : a = b) (h2 : b = 1) :
   polyrith
 
 example (x y : ℚ) (h1 : x + y = 3) (h2 : 3*x = 7) :
-    x*x*y + y*x*y + 6*x = 3*x*y + 14 :=
+    x*x*y + y*x*y + 6*x = 3*x*y + 14 := by
```

Comment:
Good catch! Are there any other similar issues in that file? (You *could*, optionally, extract all such fixes into a separate PR - that should be easy to review and merge.)

Reply:
The main problem with this test file (other than it is commented out) is that it was written in Lean 3